src/ActionGroup.php

```diff
@@ -0,0 +1,76 @@
+<?php
+declare (strict_types=1);
+
+namespace LotGD\Core;
+
+/**
+ * A grouping of navigation actions, like a submenu.
+ */
+class ActionGroup
+{
+    private $id;
+    private $title;
+    private $sortKey;
+    private $actions;
+
+    /**
+     * Create a new ActionGroup, starting with an empty set of actions.
+     * @param string $id Unique identifier for this group, in the vendor/module/group format.
+     * @param string $title Title to display to the end user. Empty string means no title.
+     * @param string $sortKey Navigation menus are displayed in the order sorted by this string.
```

Comment:
Sure, Ill change it, but I'll make it a float. The advantage of the string is that you can (almost) always insert something between two other sorting points, like between 'A' and 'B' is 'AA'.

I guess we could make it an integer and introduce a best-practice of using multiples of 10.